crates/configuration/src/shared/node.rs

```diff
@@ -206,20 +206,14 @@ impl<A> NodeConfigBuilder<A> {
 }
 
 impl NodeConfigBuilder<Initial> {
-    pub fn new(
-        default_command: Option<Command>,
-        default_image: Option<Image>,
-        default_resources: Option<Resources>,
-        default_db_snapshot: Option<AssetLocation>,
-        default_args: Vec<Arg>,
-    ) -> Self {
+    pub fn new(chain_default_context: ChainDefaultContext) -> Self {
```

Comment:
I think it makes it clearer, no ?

```suggestion
    pub fn new(context: ChainDefaultContext) -> Self {
```